stix_shifter_modules/cybereason/stix_translation/json/from_stix_map.json

```diff
@@ -0,0 +1,804 @@
+{
+  "ipv4-addr": {
+    "fields": {
+      "value": [
+        "Connection.localAddress",
+        "Connection.remoteAddress"
+      ]
+    }
+  },
+  "ipv6-addr": {
+    "fields": {
+      "value": [
+        "Connection.localAddress",
+        "Connection.remoteAddress"
+      ]
+    }
+  },
+  "network-traffic": {
+    "fields": {
+      "src_port": [
+        "Connection.localPort"
+      ],
+      "dst_port": [
+        "Connection.remotePort"
+      ],
+      "protocols[*]": [
+        "Connection.transportProtocol"
+      ],
+      "src_ref.value": [
+        "Connection.localAddress"
+      ],
+      "dst_ref.value": [
+         "Connection.remoteAddress"
+      ],
+      "src_byte_count": [
+         "Connection.aggregatedTransmittedBytesCount"
+      ],
+      "dst_byte_count": [
+        "Connection.aggregatedReceivedBytesCount",
+        "Connection.receivedBytesCount"
+      ]
+    }
+  },
+  "email-addr": {
+    "fields": {
+      "value": [
+        "User.emailAddress",
+        "User.adMail"
+      ]
+    }
+  },
+  "autonomous-system": {
+    "fields": {
+      "name": [
+        "Machine.adCompany",
+        "Machine.adOrganization",
+        "Machine.adDepartment",
+        "Machine.adOU"
+      ],
+      "number": [
+        "User.numberOfMachines"
```

Comment:
Is this the correct usage of `autonomous-system.number`? According to the spec:

> Specifies the number assigned to the AS. Such assignments are typically performed by a Regional Internet Registry (RIR)

That seems to imply some sort of identifier, but you seem to be using some sort of count.

Reply:
Removed the autonomous-system object from both from_stix_map.json and to_stix_map.json, since there is no matching attribute present in cybereason.